pyuvdata/tests/test_uvh5.py

```diff
@@ -111,9 +111,9 @@ def initialize_with_zeros(uvd, filename):
         data_dset = dgrp['visdata']
         flags_dset = dgrp['flags']
         nsample_dset = dgrp['nsamples']
-        data_dset = data
-        flags_dset = flags
-        nsample_dset = nsamples
+        data_dset = data  # noqa
```

Comment:
This one confused me and I think we need @plaplant to weigh in. I assume there's something subtle going on with hd5 file writing, because if I take out the lines that flake8 identifies as setting variables that don't get used, this entire `with ..` block goes away.

Reply:
I have this block to re-initialize the data in an HDF5 test file explicitly with zeros, so I can compare and make sure that only the partial sections that I wanted to write actually have something in them. I don't remove the test file in between tests, just overwrite the data with zeros, so I didn't want to call `create_datagroup` (the more "obvious" way to do this). I can play around and see if there's a way of doing this that makes flake8 happy, but I hope it's not too egregious if we have to leave these here (especially because this is a test file).

Reply:
I think it's ok to leave the #noqa markers, although a comment about why might be good. I think flake8 just can't tell that the `*_dset` parameters are being used -- it thinks they're defined and never used.